tf_encrypted/protocol/aby3/model/logistic_regression.py

```diff
@@ -0,0 +1,297 @@
+import tensorflow as tf
```

Comment:
I think it would be wise to move this file to the `examples` folder

Reply:
> I think it would be wise to move this file to the `examples` folder

Moving it to examples is indeed consistent with tfe's current structure. I had an ambition that we would develop more common models that users can directly use, like sklearn :). And to to honest, writing a secure model with tfe is not so straightforward at the beginning, for example, uses had to write pipelines to read data from disk in a distributed way.

Reply:
It looks similar to the model found in the `logistic` example; should we introduce a new `tf_encrypted.models` module and unite?

Reply:
yeah not a bad idea, although we would probably want to iron out the API to something we'd be happy with, as we've always thought of ours as one possible implementation and not necessarily the best or officially recommended one

Reply:
Opened https://github.com/tf-encrypted/tf-encrypted/issues/733